Fixed some styles for the installer

Ruslan-Aleev opened the pull request:
### What does it do?
Fixed some styles for the installer. Changed colors, borders, boldness, changed checkboxes via icons, made more universal error messages, deleted duplicate styles, etc.

![install_0](https://user-images.githubusercontent.com/12523676/66588845-626a3b00-eb9e-11e9-8447-b4b453d6f7a8.png)
![install_1](https://user-images.githubusercontent.com/12523676/66588847-626a3b00-eb9e-11e9-879a-95d18b86b24c.png)
![install_2](https://user-images.githubusercontent.com/12523676/66588849-6302d180-eb9e-11e9-805d-9ad786e708a3.png)
![install_3](https://user-images.githubusercontent.com/12523676/66588850-6302d180-eb9e-11e9-8e01-63b105266aac.png)

I wanted to make a design, as in sketches - https://modx3.org/updates/week-18-june-11-17-2018.
In my opinion, that green in sketches looks better.

This PR is for discussion, if the community does not accept the new green color, we can return the old green :)

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/14654
https://github.com/modxcms/revolution/pull/14507

Ibochkarev commented:
@Ruslan-Aleev Nice! Thanks

JoshuaLuckers commented on _build/templates/default/sass/installer.scss:
```diff
@@ -1,6 +1,12 @@
+$lightGray: #F6F6F6;
+$softGray: #D6D6D6;
+$blue: #4A90E2;
+$green: #14C8A0;
```
To keep the colours consistent across MODX it's better to change the values inside [_colors-and-vars.scss](https://github.com/modxcms/revolution/blob/3.x/_build/templates/default/sass/_colors-and-vars.scss)

Ruslan-Aleev replied:
I understand this, but I say that this PR is for discussion, so that global styles are not edited if the community does not accept the changes.

JoshuaLuckers reviewed (commented):
I like the new green but I'm a bit worried about the color contrast. I'm really interested in @sottwell's opinion about this.

Mark-H commented:
In current, "Next" button contrast ratio is 2.59. With this PR it's less with 2.14. For AA compatibility we need at least 4.5, which means a dark grey-ish (*or dark green hue) color rather than white. The test section seems fine though Chrome is being a pain in not showing the contrast.

Ruslan-Aleev commented:
Is contrast really important? The same WordPress does not really care about this: just a gray button on a gray background :)

Mark-H commented:
WordPress is an excellent example, thank you for bringing that up ;)

WordPress has a dedicated accessibility team with the goal "All new or updated code released in WordPress must conform with the Web Content Accessibility Guidelines 2.0 at level AA"

https://make.wordpress.org/accessibility/handbook/

And specifically: https://make.wordpress.org/accessibility/handbook/design/use-of-color/#about-color-contrast


While MODX does not currently have a dedicated team/handbook/defined goal relating to accessibility, WCAG 2/AA is a really good baseline to work towards. I'm happy if we at least _move in the right direction_.

Ruslan-Aleev commented:
Interesting, but I still won’t believe that this gray button in WP is more contrast than the green button in PR.
![ustanovka_wordpress](https://user-images.githubusercontent.com/12523676/69889440-64cd5380-130a-11ea-9071-07c747f8f3b2.png)

Mark-H commented:
[That appears to be a score of 5.34, above the 4.5 minimum for AA](https://webaim.org/resources/contrastchecker/?fcolor=656565&bcolor=F5F5F5). 

The bright green in the setup button definitely stands out more than the grays in the WP example, but contrast is about the difference between the foreground and background of the button itself. White on bright green just isn't a great fit where contrast is concerned. Making the text color a really dark green for example would much more quickly meet the minimum.

JoshuaLuckers commented:
Wrong repo guys 😋.

Ruslan-Aleev commented:
> ... but contrast is about the difference between the foreground and background of the button itself.

Thank you. Now it is more clear.

JoshuaLuckers commented:
Jokes aside, @Mark-H did a great color suggestion we should try to improve the color contrast ratio.

Ruslan-Aleev commented:
Yes, even in the current version, the manager panel colors do not pass the test,we will need to come up with something.
https://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=528738

Mark-H commented:
If completed in the next 24 hours or so this could potentially still be included in alpha1, but it's cutting it real close so I'm marking it for the next milestone.

Ruslan-Aleev commented:
There is a question with colors (https://github.com/modxcms/revolution/pull/14785/files#diff-238eaef74a8ad5b87131006e1af93bf5R4), I can remove color vars and then PR can be accepted.
**And the question of color can be raised before the next release.**

Ruslan-Aleev commented:
@Mark-H I adjusted the styles. The contrast ratio has remained the same.
I will create a separate issue and in the future we should consider new colors for better contrast.

JoshuaLuckers commented:
The Advanced Options checkboxes look disabled (the first one is, the second one isn't):

<img width="857" alt="Screen Shot 2020-01-26 at 13 00 43" src="https://user-images.githubusercontent.com/1268085/73134862-e0622d00-403b-11ea-86e5-24f6c99bf83a.png">

When the install fails, the message indicating the failure has no background

<img width="939" alt="Screen Shot 2020-01-26 at 13 02 00" src="https://user-images.githubusercontent.com/1268085/73134879-0daedb00-403c-11ea-8453-b274f094c2d4.png">

Relevant element:
<img width="1005" alt="Screen Shot 2020-01-26 at 13 02 30" src="https://user-images.githubusercontent.com/1268085/73134903-5c5c7500-403c-11ea-848f-1aec8351b534.png">

Ruslan-Aleev commented:
@cla-bot check

Ruslan-Aleev commented:
@JoshuaLuckers I changed styles for checkboxes and notifications:

![installer_1](https://user-images.githubusercontent.com/12523676/73138218-bdef0480-4079-11ea-8b78-4b47acfacd8a.png)

![installer_2](https://user-images.githubusercontent.com/12523676/73138220-be879b00-4079-11ea-91f4-e451be4ce5d1.png)

Ruslan-Aleev commented:
@cla-bot check

JoshuaLuckers commented:
Amazing work. I have some small suggestions I would love to get your feedback on:

**Welcome**
Make the selected language have a different background color (like how it is in alpha 2).
Only the thin border now indicates that it's selected, having a different background makes it more clear what language is selected.

**Options**
Consider making the selected install option (new, upgrade or advanced) also have a different background color for the same reasons as mentioned above.

**Install completed**
The checkbox has a blue background. I had the impression the checkbox was selected. This wasn't the case. Maybe use a different background color for checkboxes that are unchecked?

Ruslan-Aleev commented:
@JoshuaLuckers show visually on the screenshots what you mean, not everything is understood.

JoshuaLuckers commented:
Image of the unchecked checkbox, because of the color it might give people the impression it's checked.

<img width="601" alt="Screen Shot 2020-02-07 at 11 47 11" src="https://user-images.githubusercontent.com/1268085/74052226-d47e4f80-49d9-11ea-8904-a6b63e23c52a.png">

Current background of the selected language:
<img width="228" alt="Screen Shot 2020-02-07 at 18 45 37" src="https://user-images.githubusercontent.com/1268085/74052375-0a233880-49da-11ea-89b0-e75dc928a0e0.png">

Ruslan-Aleev committed:
Changed color to brighter

Ruslan-Aleev committed:
Fixed installer styles

Ruslan-Aleev committed:
Corrected installer "Complete" template

Ruslan-Aleev commented:
@JoshuaLuckers I corrected the styles, see gif.
For installation type I changed the color of the icons, **not the background**, because the background cannot be set for a nested span (there is a hidden checkbox, and next to the span, the border is set using the span):

![installer_color2](https://user-images.githubusercontent.com/12523676/74063714-7a49b280-4a0a-11ea-86f8-ca3a2dfcb853.gif)

I also changed the commits using force-push, but now the travis check fails, which is strange.

JoshuaLuckers reviewed (approved):
Great work!
